### DISCUSSION TITLE
Sync multiple vaults

### DISCUSSION THREAD

**Discussion by verlen64** [vrtmrz/obsidian-livesync#141] [Q&A] [answered]
Hi everyone,
I am using the plugin since 4 months now syncing my vault across 5 devices (win/ios) and it works like a charm. Thank you @vrtmrz!
Right now I am starting to use obsidian with a second vault as to divide privat from professional notes. Of course I would appreciate to sync this new vault via livesync as well. How do I have to set this up? Is it possible to use the same database-container running on my Synology NAS using a different database-name? Or will I need to run a second docker container with another instance of CouchDB?
Yours

**Comment by vrtmrz** [comment, marked as answer]
Thank you for using this plug-in! I really thank you too!

Yes, as you said, we can use the same database-container with different database-names. 
Since you have been with us for four months, I think the plug-in is well set up.
All we have to do is only set the database-name differently after coping the configuration, before the first synchronisation.

(It can be done with `Copy set-up URI`, and `Just import setting` of `Open set-up URI`)

**Reply by verlen64** [reply]
Thanks again!
I just set it up the way you suggested. I did it manually though, without using the setup wizard, but it worked anyway.
Never thought it could be that easy... :-D

**Reply by dvub** [reply]
Hey there! I am also considering switching to multiple vaults. I have a single instance of couchDB running on proxmox in an LXC. In this case, would I need another LXC, or can I get away with just the one? Thanks!
